sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorTransmitter.cs

```diff
@@ -203,25 +202,34 @@ private ExportResult HandleFailures(HttpMessage httpMessage)
                 }
             }
 
+            if (result == ExportResult.Success)
+            {
+                AzureMonitorExporterEventSource.Log.Write($"FailedToTransmit{EventLevelSuffix.Warning}", $"Error code is {statusCode}: Telemetry is stored offline for retry");
```

Comment:
We might see a message `Error code is 0`, when `_storage.SaveTelemetry(content, HttpPipelineHelper.MinimumRetryInterval);` returns success.